components/CardDropdownComponent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { styled } from "styled-components";
+import { useModalSetting } from "../hooks/useModalSetting";
+import CommonModalComponent from "./CommonModalComponent";
+
+type props = {
+  show: boolean;
+  url: string;
+};
+
+const CardDropdownComponent = ({ show, url }: props) => {
+  const [modalType, setModalType] = useState({
+    type: "",
+    title: "",
+    folderName: "",
+    linkName: "",
+  });
+  const { show: modalShow, modalOn, modalOff } = useModalSetting();
```

Comment:
요렇게 표현해주는게 더 의미가 명확할 것 같아요!
```suggestion
  const { show: modalShow, openModal, closeModal } = useModalSetting();
```